x-pack/test/functional/fixtures/kbn_archiver/spaces/enter_space.json

```diff
@@ -0,0 +1,33 @@
+{
```

Comment:
__nit:__ I don't think we need this file anymore (file isn't imported in the tests and we update config object directly via Saved Object API)?